packages/remix-server-runtime/server.ts

```diff
@@ -544,24 +544,19 @@ async function handleResourceRequest({
   }
 }
 
-function isActionRequest(request: Request): boolean {
-  let method = request.method.toLowerCase();
+function isActionRequest({ method }: Request): boolean {
   return (
-    method === "post" ||
-    method === "put" ||
-    method === "patch" ||
-    method === "delete"
+    method === "POST" ||
+    method === "PUT" ||
+    method === "DELETE" ||
+    method.toUpperCase() === "PATCH"
   );
```

Comment:
Maybe we could even simplify this to

```suggestion
    return ["POST", "PUT", "DELETE"].includes(method.toUpperCase());
```

Reply:
see new commit. no need to create/destroy the array each time